From a29062f02440d314bbe32d0f3be3dc6785af51a3 Mon Sep 17 00:00:00 2001
From: Stuart Yoder <stuart.yoder@nxp.com>
Date: Tue, 23 Aug 2016 17:13:17 -0500
Subject: [PATCH 031/107] staging: fsl-mc: move allocator private definitions
 out of mc-bus.h
Git-commit: e267dddd212e62190bcc42441ed189dfa323724c
Patch-mainline: v4.9-rc1
References: bsc#1041022

move definitions private to the allocator out of mc-bus.h and into
allocator.c

Signed-off-by: Stuart Yoder <stuart.yoder@nxp.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Alexander Graf <agraf@suse.de>

---
 drivers/staging/fsl-mc/bus/fsl-mc-allocator.c | 5 +++++
 drivers/staging/fsl-mc/include/mc-bus.h       | 5 -----
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/staging/fsl-mc/bus/fsl-mc-allocator.c b/drivers/staging/fsl-mc/bus/fsl-mc-allocator.c
index 3ebaecb..6550564 100644
--- a/drivers/staging/fsl-mc/bus/fsl-mc-allocator.c
+++ b/drivers/staging/fsl-mc/bus/fsl-mc-allocator.c
@@ -17,6 +17,11 @@
 #include "dpmcp.h"
 #include <linux/msi.h>
 
+#define FSL_MC_IS_ALLOCATABLE(_obj_type) \
+	(strcmp(_obj_type, "dpbp") == 0 || \
+	 strcmp(_obj_type, "dpmcp") == 0 || \
+	 strcmp(_obj_type, "dpcon") == 0)
+
 /**
  * fsl_mc_resource_pool_add_device - add allocatable device to a resource
  * pool of a given MC bus
diff --git a/drivers/staging/fsl-mc/include/mc-bus.h b/drivers/staging/fsl-mc/include/mc-bus.h
index 7f847e6..f192aa8 100644
--- a/drivers/staging/fsl-mc/include/mc-bus.h
+++ b/drivers/staging/fsl-mc/include/mc-bus.h
@@ -21,11 +21,6 @@
 	(strcmp((_mc_dev)->obj_desc.type, (_obj_desc)->type) == 0 && \
 	 (_mc_dev)->obj_desc.id == (_obj_desc)->id)
 
-#define FSL_MC_IS_ALLOCATABLE(_obj_type) \
-	(strcmp(_obj_type, "dpbp") == 0 || \
-	 strcmp(_obj_type, "dpmcp") == 0 || \
-	 strcmp(_obj_type, "dpcon") == 0)
-
 struct irq_domain;
 struct msi_domain_info;
 
-- 
1.8.5.6

